mimic/test/test_identity_objects.py

```diff
@@ -0,0 +1,302 @@
+from __future__ import absolute_import, division, unicode_literals
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from mimic.model.identity_objects import (
+    bad_request,
+    not_found,
+    forbidden
+)
+from mimic.test.dummy import (
+    ExampleEndPointTemplate,
+    make_example_external_api
+)
+
+
+class YetToBeDone(SynchronousTestCase):
+    """
+    Test that are a placeholder for necessary functionality
+    that will be needed when the full functionality is implemented.
+    """
+
+    def test_stuff_todo(self):
+        """
+        Temporary for code-coverage until end-points are
+        implemented that will actually use these.
+        """
+        class reqMock(object):
+            def setResponseCode(self, code):
+                pass
+
+        bad_request("testing bad request", reqMock())
+        not_found("testing not found", reqMock())
+        forbidden("testing forbidden", reqMock())
```

Comment:
I'll have that in the next PR; this PR doesn't contain the actual RESTful API endpoints, it's just infrastructure. The next PR will have the endpoints that all this infrastructure supports. Thus why I'd like to just leave it as is now, and then use or remove in the next PR.